package.json

```diff
@@ -12,7 +12,7 @@
     "prepush": "npm test",
     "build": "webpack --config ./webpack.prod.config.js --progress --colors",
     "start": "npm run build && NODE_ENV=production node server.js",
-    "start:dev": "webpack --config ./webpack.config.js --mode=development && node server.js"
+    "start:dev": "webpack-dev-server --config ./webpack.config.js --mode=development && node server.js"
```

Comment:
Why use webpack-dev-server and node server both?

Reply:
thanks for this comment, fixed that. Now, I have a better understanding of this.